lips/lip-20.md

```diff
@@ -0,0 +1,595 @@
+---
+lip: 20
+title: Libra On-Chain Currency Management
+authors: Tim Zakian, Sam Blackshear
+status: Draft
+type: Informational
+created: 08/28/2020
+---
+
+---
+# Summary
+---
+
+This LIP describes the conceptual model and implementation of currencies on the Libra
+blockchain. Currencies in Libra are represented statically at the type level, each
+currency with a unique Move type. A type is treated as a currency in Libra
+if and only if it has previously been registered as a currency on-chain. Once a currency
+is registered it remains registered forever.
+
+Every currency that is registered on-chain has metadata
+associated with it at the time of registration. That metadata includes the currency code, value
+held on-chain in the currency, and other relevant information[[1]](#registration_and_metadata). One of the most
+important properties held in a currency's metadata is whether it is a Single Currency Stablecoin
+(SCS), i.e., a fiat currency, or a synthetic currency that may consist of one or
+multiple other currencies registered on-chain, e.g., ≋LBR.
+
+On-chain, all assets with a value of `x` in a registered currency `C` are held as a "Libra Coin" resource
+of type [`Libra<C>`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#resource-libra)
+with its `value` field set to `x`. Every Libra Coin has a certain set of
+operations that may be performed on it and other coins of the same
+currency[[2]](#general_currency_operations), and every registered currency possesses a
+specific set of metadata about itself[[3]](#metadata_spec) that may be
+queried[[4]](#currency_ops). Additionally, none of the operations detailed in the rest of
```

Comment:
This sentence has too many negatives to grok easily :). Perhaps: "All operations must be be performed on a registered currency type"?